ui/component/notification/view.jsx

```diff
@@ -57,6 +62,24 @@ export default function Notification(props: Props) {
     urlParams.append('lc', notification_parameters.dynamic.hash);
   }
 
+  const notificationTitle = notification_parameters.device.title;
+  const titleSplit = notificationTitle.split(' ');
+  let creatorName;
+  titleSplit.map((s) => {
+    if (!creatorName && s.includes('@')) {
+      creatorName = s;
+    }
+  });
+  const title = titleSplit.map((message) => {
+    if (creatorName === message) {
+      return (
+      <UriIndicator uri={notification_parameters.dynamic.comment_author} link />
+      );
+    } else {
+      return <LbcMessage>{__(' %message% ', { message })}</LbcMessage>;
+    }
+  });
```

Comment:
@jessopb, I believe this block is trying replace the channel name with something that's clickable.  I recall this issue also affects livestream chat and maybe other places.  Do you have a recommendation on how to address this issue in general?

The above breaks up the string.  This will break any hopes of localizing the notifications page in the future.
  - <img src="https://user-images.githubusercontent.com/64950861/127441224-22c8ef63-71e6-4938-8c3e-fd4692bae5a4.png" width="400">
  - <img src="https://user-images.githubusercontent.com/64950861/127441451-c35e5f70-8657-4196-b502-9948032f7d01.png" width="400">

Maybe something like how `LbcMessage` does the replacement will be a cleaner approach?

But anyway, since Commentron doesn't support a i18n-friendly format yet, this solution is still better than current state.

Reply:
Or maybe something like this, which also fixes the i18n problem:
```
  const commentedOnClaimMsg = (
    <I18nMessage
      tokens={{
        comment_author: <UriIndicator uri={notification_parameters.dynamic.comment_author} link />,
        claim_title: notification_parameters.dynamic.claim_title,
      }}
    >
      %comment_author% commented on %claim_title%
    </I18nMessage>
  );
```